app/views/exercises/show.html.erb

```diff
@@ -1,9 +1,13 @@
 <%= render_runner_assets @exercise.language, :layout %>
 
 <%= content_for :breadcrumbs do %>
-    <%= breadcrumbs @exercise %>
+  <%= breadcrumbs @exercise %>
 <% end %>
 
+<%= content_for :next_exercise_button do %>
+  <span></span>
```

Comment:
Instead of using `content_for` here, why not just using `if @embedded_mode ... else ....` in `_results_button.html.erb`. I think that will be simpler and more intention revealing, won't it?